tests/shared/src/test/scala/cats/tests/KleisliSuite.scala

```diff
@@ -89,6 +89,12 @@ class KleisliSuite extends CatsSuite {
            SerializableTests.serializable(CommutativeMonad[Kleisli[Id, Int, *]])
   )
 
+  {
+    implicit val K: Align[Kleisli[Id, MiniInt, *]] = Kleisli.catsDataAlignForKleisli[Id, MiniInt]
```

Comment:
For some reason this is required for scala 2.12.x, as it fails to find the implicit Align for Kleisli on its own. Is there another way to do this?

Reply:
I guess it must be an inference issue. /shrug
```
[error] /workspace/cats/tests/shared/src/test/scala/cats/tests/KleisliSuite.scala:93:51: could not find implicit value for evidence parameter of type cats.Align[[γ$23$]cats.data.Kleisli[[A]A,cats.laws.discipline.MiniInt,γ$23$]]
[error]     checkAll("Kleisli[Id, MiniInt, *]", AlignTests[Kleisli[Id, MiniInt, *]].align[Int, Int, Int, Int])
[error]                                                   ^
```